### ISSUE TITLE
what is the massive_epoch_240 checkpoint?

### ISSUE THREAD

**Issue by rishabh063** [ZhengPeng7/BiRefNet#22]
hey saw i new checkpoint in stuff what is it?

**Comment by ZhengPeng7** [comment]
Aha, thx for the continual attention :)
I did a similar thing in [InSPyReNet](https://github.com/plemeri/InSPyReNet/blob/main/docs/model_zoo.md#-trained-with-massive-sod-datasets-with-lr-384-x-384-and-hr-1024-x-1024-not-in-the-paper-for-real-world-scenarios) to train the model with massive datasets. In my setting, the massive training set is [DIS-TR, DIS-TEs] + [HRSOD-TR, HRSOD-TE] + [UHRSD-TR, UHRSD-TE] + [[HRS10K-TR, HRS10K-TE](https://github.com/DrowsyMon/RMFormer)], and the DIS-VD is left as validation set. You can find it in [config.py](https://github.com/ZhengPeng7/BiRefNet/blob/bda8a1ab3e0dd4277cfcb3da45f8843ad05e4bfe/config.py#L17).

**Comment by rishabh063** [comment]
Thanks for the training , how are the results when compared to closed source solutions ?

i made a quick colab file for people to try this repo in a easier way for single image inference ( i have written this 5-10 times but always loose it edit the same colab like a noob) 

attaching the link : -   https://colab.research.google.com/drive/1aCBF9THgyHHxJwoIYfkYfHOjv6m0y1A8?usp=sharing.

**Comment by ZhengPeng7** [comment]
Good! Thanks. I'll add the link to it in the README. 
About the model trained on massive data, it's not better than the BiRefNet-DIS on DIS-VD. But I think it should work better on the wild images.
